.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+jobs:
+  release:
+    name: Create new release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            filename_suffix: "${{ github.ref_name }}-linux-amd64"
+    steps:
+      - uses: actions/checkout@v4
+      - run: gh release create ${{ github.ref }} --draft --title "Uhyve ${{ github.ref_name }}"
+      - uses: dtolnay/rust-toolchain@stable
```

Comment:
> if: startsWith(matrix.os, 'ubuntu')

from https://github.com/mkroening/edu-sync/blob/1a70834a82c69ed284b2fb620387380c1ef6704f/.github/workflows/release.yml#L24C1-L24C42 is not present.

Reply:
Indeed it does, but I think that the next person to work on this workflow (most importantly, future me) would be in for a *very* frustrating surprise if they were to add more matrix elements. ^^